plugins/zscaler/unit_test/expected/success.json.exp

```diff
@@ -0,0 +1,3 @@
+{
+  "success": true
+}
```

Comment:
FYI, remember to add new line at the end of files 😄

Reply:
Updated. Thanks for your suggestion.